modules/button/css/lib/icon-button.scss

```diff
@@ -0,0 +1,244 @@
+
+      &:not([disabled]):not(.wdc-btn-disabled) {
+        &.wdc-btn-hover,
+        &:hover {
+          background-color: $wdc-btn-bg-icon-hover;
+        }
+
+        &.wdc-btn-active,
+        &:active {
+          background-color: $wdc-btn-bg-icon-active;
+        }
+
+        &:focus,
+        &.wdc-btn-focus {
+          background-color: $wdc-btn-bg-icon-focus;
+
+          &.wdc-btn-hover,
+          &:hover {
+            background-color: $wdc-btn-bg-icon-hover;
+
+            &.wdc-btn-active,
+            &:active {
+              background-color: $wdc-btn-bg-icon-active;
+            }
+          }
+        }
+      }
+
+      &.wdc-btn-disabled,
+      &:disabled {
+        background-color: $wdc-btn-bg-icon-disabled;
+
+        .wd-icon-accent,
+        .wd-icon-fill {
+          fill: $wdc-btn-color-icon-disabled;
+        }
+      }
+    }
+
+    &-inverse {
+      background-color: transparent;
+      .wd-icon-accent,
+      .wd-icon-fill {
+        fill: $wdc-btn-bg-icon;
+      }
+
+      &.wdc-btn-hover:not([disabled]):not(.wdc-btn-disabled),
+      &:hover:not([disabled]):not(.wdc-btn-disabled) {
+        background-color: rgba(0, 0, 0, 0.2);
+        .wd-icon-accent,
+        .wd-icon-fill {
+          fill: $wdc-btn-bg-icon;
+        }
+      }
+
+      &.wdc-btn-active:not([disabled]):not(.wdc-btn-disabled),
+      &:active:not([disabled]):not(.wdc-btn-disabled) {
+        background-color: rgba(0, 0, 0, 0.2);
+
+        .wd-icon-accent,
+        .wd-icon-fill {
+          fill: $wdc-btn-bg-icon;
+        }
+
+        &.wdc-btn-hover:not([disabled]):not(.wdc-btn-disabled),
+        &:hover:not([disabled]):not(.wdc-btn-disabled) {
+          background-color: rgba(0, 0, 0, 0.4);
+
+          .wd-icon-accent,
+          .wd-icon-fill {
+            fill: $wdc-btn-bg-icon;
+          }
+        }
+      }
+
+      &.wdc-btn-disabled,
+      &:disabled {
+        background-color: transparent;
+
+        .wd-icon-accent,
+        .wd-icon-fill {
+          fill: $wdc-btn-bg-icon-disabled;
+        }
+      }
+
+      &:not([disabled]):not(.wdc-btn-disabled) {
+        &.wdc-btn-hover,
+        &:hover {
+          background-color: rgba(0, 0, 0, 0.2);
+        }
+
+        &.wdc-btn-active,
+        &:active {
+          background-color: rgba(0, 0, 0, 0.2);
+        }
+
+        &:focus,
+        &.wdc-btn-focus {
+          background-color: transparent;
+
+          &.wdc-btn-hover,
+          &:hover {
+            background-color: rgba(0, 0, 0, 0.2);
+          }
+          &.wdc-btn-active,
+          &:active {
+            background-color: rgba(0, 0, 0, 0.4);
+          }
+        }
+      }
+    }
+
+    &-square,
+    &-square-filled {
+      border-radius: $wdc-btn-border-radius-icon;
+    }
+
+    border-radius: 999;
+
+    &:not([disabled]):not(.wdc-btn-disabled) {
+      &.wdc-btn-active,
+      &:active {
+        box-shadow: 0 0 0 2px white, 0 0 0 4px $wdc-btn-border-active;
```

Comment:
Just calling this out. I noticed this is the only one in CKR that uses `0 0 0 2px $wdc-btn-border-active`. Everything else has the 2px white ring. This is inconsistent right now, so we may want to update the React version. **Follow up needed**

Reply:
Another note: We should probably support the toggled states as well, but I think we can add that later down the road.

Reply:
Focus outline update captured here: https://github.com/Workday/canvas-kit/issues/97